code/modules/mining/satchel_ore_box.dm

```diff
@@ -10,7 +10,7 @@
 	pressure_resistance = 5*ONE_ATMOSPHERE
 
 /obj/structure/ore_box/attackby(obj/item/W, mob/user, params)
-	if (istype(W, /obj/item/stack/ore))
+	if (istype(W, /obj/item/stack/ore) || istype(W, /obj/item/boulder))
 		user.transferItemToLoc(W, src)
```

Comment:
`transferItemToLoc()` can return FALSE and fail. Probably early return if that happens